### PR TITLE
chore(setup): Added additional metainfo

### PR DESCRIPTION
**Pull request by GMishx**
1. Added metainfo like `license` and `platform` to setup.py.
2. Added `MANIFEST.in` to remove test data from package.
3. New exception `NotSupportedExtension` to handle files with unknown extensions.
4. Remove `cwd` infront of filename. (this was causing nirjas to not run on other directories).

Signed-off-by: Gaurav Mishra <mishra.gaurav@siemens.com>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -37,22 +39,24 @@
...
-    name='Nirjas', 
-    version='0.0.2',  
-    description='A Python library to extract comments and source code out of your file(s)',  
-    long_description=long_description,  
-    long_description_content_type='text/markdown',  
-    url='https://github.com/fossology/nirjas',  
-    author='Ayush Bhardwaj, Kaushlendra Pratap',  
-    author_email='classicayush@gmail.com, kaushlendrapratap.9837@gmail.com',  
+    name='Nirjas',
+    version='0.0.4',
```
Comment (hastagAB) [inline setup.py:43]: We haven't released v0.0.3 yet. After this PR we are planning to release Nirjas v0.0.3

**Comment by GMishx** [comment]
> It would be really beneficial if we could add Travis to Nirjas.

I am working on that right now. Maybe you can create an issue and assign it to me.

File: setup.py (hunk 1 above)
Comment (GMishx) [reply setup.py:43]: I saw the version update in #10 to 0.0.3 so I thought you will publish the package before current branch gets merged. If the changes looks good to you, I will revert to 0.0.3 and push the changes.

Comment (hastagAB) [reply setup.py:43]: Yes Please, We haven't made a release yet. I will publish it after merging this PR.

**Comment by hastagAB** [comment]
> > It would be really beneficial if we could add Travis to Nirjas.
> 
> I am working on that right now. Maybe you can create an issue and assign it to me.

Sure, Thanks! 
I'm creating an issue.

**Pull request merged**